src/sync.rs

```diff
@@ -260,7 +260,7 @@ impl<K: Eq + Hash, V: Copy> FrozenMap<K, V> {
     /// new entry in the map for that key and returns a
     /// reference to the given value.
     ///
-    /// Existing values are never overwritten.
+    /// This is safe because existing values are never overwritten.
```

Comment:
Ah, I meant in the function body right before the entry line, saying that this function must not ever overwrite and usage of entry() + or_insert ensures that

Reply:
The doc comment on the function itself need not make claims about safety

Reply:
Oops, I see now. Do the new changes look better?